### ISSUE TITLE
[site:new] Fails if Drupal version is specified

### ISSUE THREAD

**Issue by sshrestha-navarts** [hechoendrupal/DrupalConsole#1113]
# Problem/Motivation

Trying to install a new Drupal 8  site (specific version) from the command line using DrupalConsole

# How to reproduce

`drupal site:new test 8.0.0-rc2` fails as it tries to download drupal-8.x-8.0.0-rc2.tar.gz 

Drupal Console version 0.9.4

![screen shot 2015-10-28 at 1 25 11 pm](https://cloud.githubusercontent.com/assets/13104494/10797084/598b8b2c-7d77-11e5-927b-58f076441b68.png)

**Comment by VasquezPonte** [comment]
PHP version: 5.6
Drupal Console version: **0.9.5**

I cannot reproduce the issue on my environment.

`drupal  site:new  test  8.0.0-rc2`

```
[+] Downloading Drupal 8.0.0-rc2
[+] Extracting files for Drupal 8.0.0-rc2
[+] Drupal 8.0.0-rc2 was downloaded in directory test
```

**Comment by sauravshrestha** [comment]
@VasquezPonte thanks!

It seems I was on *0.9.4* and doing a `drupal self-update` fixed it for me.

**Comment by jmolivas** [comment]
This must be fixed on v0.9.5 https://github.com/hechoendrupal/DrupalConsole/releases/tag/0.9.5 we replace Buzz library with Guzzle to solve the issue.

**Comment by sauravshrestha** [comment]
Its fixed now. The problem I was seeing has been addressed by this commit https://github.com/jmolivas/DrupalAppConsole/commit/5b3531b32991d6823307e8493e8b66c0e2b4c12a

**Comment by jmolivas** [comment]
Fixed on v0.9.5 https://github.com/hechoendrupal/DrupalConsole/releases/tag/0.9.5
